reference/forms/types/collection.rst

```diff
@@ -363,6 +364,31 @@ rendering your form, having the entire "form row" may be easier for you.
 For details on how to actually use this option, see the above example as
 well as :ref:`cookbook-form-collections-new-prototype`.
 
+prototype_data
+~~~~~~~~~~~~~~
+
+.. versionadded:: 2.8
+    The ``prototype_data`` option was introduced in Symfony 2.8.
+
+**type**: ``mixed`` **default**: ``null``
+
+Allows you to define specific data for the prototype. Each new row added will
+initially contain the data set by this option. By default, the data configured
+for all entries with the ``entry_options`` option will be used.
+
+.. code-block:: php
+
+    use Symfony\Component\Form\Extension\Core\Type\CollectionType;
+    use Symfony\Component\Form\Extension\Core\Type\TextType;
```

Comment:
Right, I made another commit to do that change in all examples, so you can easily revert it in 3.0+ branches. Thanks!